pkg/apis/rook.io/v1/types.go

```diff
@@ -124,7 +124,8 @@ type StorageClassDeviceSet struct {
 	Config               map[string]string          `json:"config,omitempty"`               // Provider-specific device configuration
 	VolumeClaimTemplates []v1.PersistentVolumeClaim `json:"volumeClaimTemplates,omitempty"` // List of PVC templates for the underlying storage devices
 	Portable             bool                       `json:"portable,omitempty"`             // OSD portability across the hosts
-	TuneSlowDeviceClass  bool                       `json:"tuneDeviceClass,omitempty"`      // TuneSlowDeviceClass Tune the OSD when running on a slow Device Class
+	TuneSlowDeviceClass  bool                       `json:"tuneSlowDeviceClass,omitempty"`  // TuneSlowDeviceClass Tune the OSD when running on a slow Device Class
```

Comment:
You cannot just change the Spec of something in production. Let's keep `tuneDeviceClass` even if not ideal, otherwise, it's a breaking change.

Reply:
Reverted.